### PR TITLE
ABRT is dead - long live Retrace

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

**Comment by xsuchy** [comment]
This is ready for review. But I want to postpone the publication for some time.

File: _posts/2020-11-01-abrt-is-dead-long-live-retrace.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "ABRT is dead - long live Retrace"
+date:   2020-11-01 12:00:00 +0100
+author: Miroslav Suchý
+visible: 1
+categories:
+---
+
+A few weeks ago, I noted that [we are looking for a new name for ABRT](/2020/09/09/abrt-new-name/). After a discussion, we come to the name **Retrace**.
```
Comment (michalfabik) [inline _posts/2020-11-01-abrt-is-dead-long-live-retrace.markdown:10]: after _some_ discussion

File: _posts/2020-11-01-abrt-is-dead-long-live-retrace.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+It has several benefits:
+
+ * It is not an entirely new name. Part of the ABRT is [retrace-server](https://github.com/abrt/retrace-server/). We have a public server called [retrace](https://retrace.fedoraproject.org/).
+ * It is easy to pronounce.
+ * It reflects what we are doing: finding and reporting tracebacks.
+
+What will be the names of components:
+
+ * "abrt" is going to be renamed to "retrace".
+ * "abrt analytics" (formerly faf) will be renamed to "retrace analytics".
```
Comment (michalfabik) [inline _posts/2020-11-01-abrt-is-dead-long-live-retrace.markdown:21]: Maybe mention abrt-java-connector for the sake of completeness?

**Review by michalfabik — COMMENTED** [review]
I think you should elaborate a little on the scope of the changes - i.e. changes in the frontend, changes to executable names, package names, names of GitHub repos, changes in the code itself etc.

**Comment by xsuchy** [comment]
Updated. And we need to choose the new name of the organization as retrace is taken by a user. 

@retrace you do not have an email assigned to your account so this is the only way to contact you. I see you do not have any visible activity on GitHub. Is that any chance to close/rename your account so we can use the @retrace as an organization for our project? If this gets to you please contact me on "msuchy@redhat.com".

**Comment by msrb** [comment]
Since we don't plan to rename the project now, I am gonna close this PR. We can reopen it in future if needed.

**Pull request closed without merging**